spinnaker-monitoring-daemon/spinnaker-monitoring/spectator_client.py

```diff
@@ -211,19 +211,29 @@ def get_source_catalog(options):
   if _cached_registry_timestamp == timestamp:
     return _cached_registry_catalog
 
+  spectator_options = options.get('spectator', {})
   logging.info('Updating catalog from %s at %ld', registry_dir, timestamp)
   catalog = {}
   for source in glob.glob(os.path.join(registry_dir, '*.yml')):
     name = os.path.splitext(os.path.basename(source))[0]
-    logging.info('loading %s', source)
+    if spectator_options.get('use_base_service_name_only'):
```

Comment:
Shouldn't this be `spectactor_options['use_base_service_name_only']` ? 

After trying to enable this with 
```
spinnaker-monitoring-local.yml: |-
        spectator: |-
          use_base_service_name_only: true
```
I get the error below:
```
23:23:07 INFO Updating catalog from /opt/spinnaker-monitoring/registry at 1555543298
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/usr/local/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/opt/spinnaker-monitoring/bin/__main__.py", line 190, in <module>
    main(search_path)
  File "/opt/spinnaker-monitoring/bin/__main__.py", line 166, in main
    options['command'], options, all_command_handlers)
  File "/opt/spinnaker-monitoring/bin/command_processor.py", line 174, in process_command
    entry.process_commandline_request(options)
  File "/opt/spinnaker-monitoring/bin/server_handlers.py", line 165, in process_commandline_request
    metric_service_list = self.make_metric_services(options)
  File "/opt/spinnaker-monitoring/bin/server_handlers.py", line 135, in make_metric_services
    service_list.append(factory(options, self.command_handlers))
  File "/opt/spinnaker-monitoring/bin/prometheus_service.py", line 360, in __call__
    return PrometheusMetricsService(options)
  File "/opt/spinnaker-monitoring/bin/prometheus_service.py", line 189, in __init__
    self.__catalog = spectator_client.get_source_catalog(options)
  File "/opt/spinnaker-monitoring/bin/spectator_client.py", line 253, in get_source_catalog
    if spectator_options.get('use_base_service_name_only'):
AttributeError: 'str' object has no attribute 'get'
```